images/.hadolint.yaml

```diff
@@ -6,8 +6,12 @@ ignored:
   - DL3006
   # Pin versions in apt get install: https://github.com/hadolint/hadolint/wiki/DL3008
   - DL3008
+  # Avoid additional packages by specifying --no-install-recommends: https://github.com/hadolint/hadolint/wiki/DL3015
+  - DL3015
   # Pin versions in apk add: https://github.com/hadolint/hadolint/wiki/DL3018
   - DL3018
+  # Do not use apt as it is meant to be an end-user tool, use apt-get or apt-cache instead: https://github.com/hadolint/hadolint/wiki/DL3027
+  - DL3027
```

Comment:
I cleaned these up in the past (#821 handled both), and I'd prefer to not globally regress.  If we need to, we can allowlist these for a specific file, IIRC.

Reply:
Good catch, yeah I'll remove it